### DISCUSSION TITLE
Images should be centered by default (instead of left justify).

### DISCUSSION THREAD

**Discussion by bershatsky** [adityatelange/hugo-PaperMod#602] [Ideas]
Any way to make this happen?

**Comment by kdkasad** [comment]
Yes. I don't think it's documented, but see my example page: https://github.com/kdkasad/hugo-PaperMod/blob/exampleSite/content/test-pages/figure-align/index.md

**Reply by adityatelange** [reply]
Yes this is documented
https://github.com/adityatelange/hugo-PaperMod/wiki/FAQs#centering-image-in-markdown

**Reply by bershatsky** [reply]
Thanks!
